pkg/model/application.proto

```diff
@@ -45,6 +45,8 @@ message Application {
     string platform_provider = 15;
     // The names of deploy taget where to deploy this application.
     repeated string deploy_targets = 16;
+    // The name of plugin used to deploy this application.
+    string plugin = 17;
```

Comment:
it's not a single plugin which deploys an application but it's a list of plugins do

Reply:
@khanhtc1202 
Thanks, fixed it on [0c4fc0d](https://github.com/pipe-cd/pipecd/pull/5457/commits/0c4fc0d2de55fdc432b3c24bdd9b30b82d613311)